{{cookiecutter.github_repository}}/Makefile

```diff
@@ -71,26 +80,26 @@ clean:  ## Remove all temporary files like pycache
 # == Django Helpers
 # ===================================================
 djrun: install  ## Start Django server locally
-	${ENV_PREFIX}python manage.py runserver
+	${ENV_PREFIX}poetry run python manage.py runserver
 
 
 test: ARGS=--pdb --cov  ## Run all the tests
 test: lint
-	${ENV_PREFIX}pytest $(ARGS)
+	${ENV_PREFIX}poetry run pytest $(ARGS)
 
 djmm:  ## Create Django migrations
-	${ENV_PREFIX}python manage.py makemigrations
+	${ENV_PREFIX}poetry run python manage.py makemigrations
 
 djmigrate:  # Run Django migrations
-	${ENV_PREFIX}python manage.py migrate
+	${ENV_PREFIX}poetry run python manage.py migrate
 
 djurls:  ## Displays all the django urls
-	${ENV_PREFIX}python manage.py show_urls
+	${ENV_PREFIX}poetry run python manage.py show_urls
 
 shell:  ## Enter the django shell
-	${ENV_PREFIX}python manage.py shell_plus
+	${ENV_PREFIX}poetry run python manage.py shell_plus
 
-docs: venv  ## Start documentation server locally
+docs: virtualenv  ## Start documentation server locally
 	${ENV_PREFIX}python -m pip install -r requirements/docs.txt
```

Comment:
If I'm not wrong `requirements/docs` won't be there now.

Reply:
Fixed this everywhere.